File: Introductory Problems/Bit Strings/bit_strings.cpp
#include<bits/stdc++.h>
using namespace std;
#define ll long long int

void solve(){
	ll n, ans = 1, mod = 1e9+7;
	cin>>n;
	for(int i=0;i<n;i++){
		ans = (ans*2)%mod;
	}
	cout<<ans;
	return;
	
}

int main(){
	long long testCases;
	testCases = 1;
	// cin>>testCases;
	while(testCases--){
		solve();
	}
	return 0;
}

File: Introductory Problems/Bit Strings/bitStrings.java
import java.util.Scanner;

public class bitStrings {
    private static void solve(){
        long n, ans = 1, mod = 1000000007;
        Scanner scanner = new Scanner(System.in);
        n = scanner.nextInt();
        for(int i=0;i<n;i++){
            ans = (ans*2)%mod;
        }
        System.out.print(ans);
        scanner.close();
        return;
    }
    public static void main(String args[]){
        solve();
    }
}

